shoulder/filter/__init__.py

```diff
@@ -40,8 +40,11 @@
 # registers = filters["filter_name"].filter_inclusive(registers)
 
 filters = OrderedDict()
+filters["aarch32"] = aarch32.AArch32RegisterFilter()
+filters["aarch64"] = aarch64.AArch64RegisterFilter()
 filters["activity_monitor"] = activity_monitor.ActivityMonitorRegisterFilter()
 filters["deprecated"] = deprecated.DeprecatedRegisterFilter()
+filters["external"] = external.ExternalRegisterFilter()
```

Comment:
With the addition of the AArch32, AArch64, and External filters, `filter_all()` now removes all registers. That might not be an issue since we aren't really using that function anymore, but that function isn't really that useful anymore unless you actually remove filters from the `filters` dictionary.

Reply:
Nice, good catch. I'm thinking that ```filter_all``` doesn't have much utility anymore after expanding our model of registers/access mechanisms.

I vote we remove it!

Reply:
I agree, let's just toss it. I'd rather have a wide diversity of filters that we can choose from. If you want to remove it in this PR, this should be good to be merged.

Reply:
I'll delete it in a future PR along with a bunch of other dead-code deletion. Sounds good, thanks for the review!